person.rb

```diff
@@ -1,15 +1,16 @@
 # frozen_string_literal: true
 
-require 'securerandom'
 require_relative 'nameable'
 require_relative 'rental'
 
 class Person < Nameable
+  @@id_counter = 0
   def initialize(age, name = 'Unknown', parent_permission = true)
     raise ArgumentError, 'Age cannot be nil.' if age.nil?
 
     super()
-    @id = SecureRandom.uuid
+    @id = @@id_counter
+    @@id_counter += 1
     @name = name
     @age = age
     @parent_permission = parent_permission
```

Comment:
- [ ] Good job so far, however, the app is throwing an error in an attempt to create a person of type _teacher_. When a specialization of the _teacher_ person is entered then the app crashes. Let's improve upon this aspect as seen below:

**Error whiles creating a teacher person:**
![image](https://user-images.githubusercontent.com/12745474/225798726-63e0ec0a-8293-458f-bba2-dee0aad7ffca.png)